app/_config/docs-repositories.yml

```diff
@@ -162,3 +162,7 @@ RefreshMarkdownTask:
       - silverstripe/silverstripe-documentconverter
       - silverstripe-documentconverter
       - master
+    -
+      - silverstripe/ckan-registry
```

Comment:
This looks like a GitHub slug to me - maybe should be `silverstripe/silverstripe-ckan-registry`?

Reply:
Will update.